File: Binary Tree Inorder Traversa/Binary Tree Inorder Traversa.py
# Definition for a binary tree node.
# class TreeNode(object):
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right

class Solution(object):
    def inorderTraversal(self, root):
        """
        :type root: TreeNode
        :rtype: List[int]
        """
        
        def travarse(root):
            if root:
                travarse(root.left)
                self.result.append(root.val)
                travarse(root.right)
        
        self.result = []
        travarse(root)
        
        return self.result


        

File: Binary Tree Inorder Traversa/Binary Tree Inorder Traversa.php
<?php

/**
 * Definition for a binary tree node.
 * class TreeNode {
 *     public $val = null;
 *     public $left = null;
 *     public $right = null;
 *     function __construct($val = 0, $left = null, $right = null) {
 *         $this->val = $val;
 *         $this->left = $left;
 *         $this->right = $right;
 *     }
 * }
 */
class Solution {

    /**
     * @param TreeNode $root
     * @return Integer[]
     */

    var $result = [];

    function travarse($root){
        if($root != null){
            $this->travarse($root->left);
            array_push($this->result, $root->val);
            $this->travarse($root->right);
        }
    }

    function inorderTraversal($root) {        
        $this->travarse($root);
        return $this->result;
    }

}

?>